src/Parser.php

```diff
@@ -34,7 +34,6 @@ public function __construct($key, $configPath = null)
             $configPath = __DIR__.'/.env';
         }
         $this->setConfigPath($configPath);
-        $this->contents = $this->loadFile($configPath);
```

Comment:
we don't just drop this unnecessary dynamic property but also remove a unnecessary config pah file read